v0/destinations/ga/transform.js

```diff
@@ -140,6 +140,7 @@ function processPageViews(message, destination) {
         if (search && includeSearch) {
           documentPath += search;
         }
+        documentPath = encodeURIComponent(documentPath);
```

Comment:
@sauravlal-rudderstack can you put some destination API reference for this

Reply:
destination API reference: https://developers.google.com/analytics/devguides/collection/protocol/v1/reference